nginx_proxy/data/run.sh

```diff
@@ -24,6 +27,30 @@ if [ ! -f "$SNAKEOIL_CERT" ]; then
     openssl req -x509 -nodes -days 3650 -newkey rsa:2048 -keyout $SNAKEOIL_KEY -out $SNAKEOIL_CERT -subj '/CN=localhost'
 fi
 
+if [ "$CLOUDFLARE" == "true" ]; then
+    sed -i "s|#include /data/cloudflare.conf;|include /data/cloudflare.conf;|" /etc/nginx.conf
+    # Generate cloudflare.conf
+    if [ ! -f "$CLOUDFLARE_CONF" ]; then
```

Comment:
At last, this list will be never updated after first creating. Is that okay or do we want update the list of IPs on every start but do not fails if we are not able to update it?

Reply:
I checked the list with the list using Wayback Machine, back up to three years ago, and essentially one entry got removed, which means using the list from three years ago has one unnecessary entry, but that’s about it. So i don't see a need to update it on every run.